.github/scripts/Tests/lesson-promotion-manifest.Tests.ps1

```
@@ -728,6 +733,107 @@ description: "Another skill. Use when reconciling a promoted roster against its
+        #       chunk 3 did) reds under equality;
+        #   (a) the tree carried no unmanifested file, so the test never demonstrated the silence
+        #       its own drift message asserts - that an uncovered skill's orphan scan is skipped.
+        # This pins all three.
+        $t = script:New-ScratchTree {
+            param($c)
+            # An orphan the covering run must report and the uncovered run must MISS.
+            Set-Content -LiteralPath (Join-Path $c.Dir 'skills/demo-skill/references/orphan.md') `
+                -Encoding utf8 -Value '# Not in the manifest'
+        }
+
+        $args = @{ RepoRoot = $t.Dir; ManifestPath = $t.ManifestPath
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)